navigation/navigation.html

```diff
@@ -0,0 +1,98 @@
+            <a class="nav-navigator-item" href="#">Home</a>
+            <a class="nav-navigator-item" href="#">Documentation</a>
+        </div>
+
+
+        <button class="nav-github-holder">  <a  href="#"><i class="fab fa-github">  View on Github</i></a></button>
+
+    </nav>
+    <div class="carbon-code">
+        <iframe src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C0%29&t=seti&wt=sharp&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%253Cnav%2520class%253D%2522navigation-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522nav-logo-and-title%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522nav-logo%2522%2520src%253D%2522...%2522%2520alt%253D%2522%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522navigation-title%2522%253E%2520CodesUp%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522nav-navigators%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520class%253D%2522nav-navigator-item%2522%2520href%253D%2522%2523%2522%253EHome%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520class%253D%2522nav-navigator-item%2522%2520href%253D%2522%2523%2522%253EDocumentation%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522nav-github-holder%2522%253E%2520%2520%253Ca%2520%2520href%253D%2522%2523%2522%253E%253Ci%2520class%253D%2522fab%2520fa-github%2522%253E%2520%2520View%2520on%2520Github%253C%252Fi%253E%253C%252Fa%253E%253C%252Fbutton%253E%250A%250A%2520%2520%2520%2520%253C%252Fnav%250A%250A"
+            style=" height: 300px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+      </iframe>
+    </div>
+    <!-- ecommerce navigation bar -->
+    <h2 class="component-heading"> e-commerce Navigation</h2>
+    <div class="component-description">e-commerce Navigation is used for e-commerce app </div>
+    <nav class="nav-container">
+        <div class="nav-logo-and-title">
+            <img class="nav-logo" src="images/logo.png" alt="">
+            <div class="nav-title"> DressUp</div>
+        </div>
+
+        <div class="nav-navigators">
+            <a class="nav-navigator-item" href="#">Home</a>
+            <a class="nav-navigator-item" href="#">Kids</a>
+            <a class="nav-navigator-item" href="#">Women</a>
+            <a class="nav-navigator-item" href="#">Men</a>
+
+        </div>
+
+        <div class="nav-search">
+            <span class="material-icons-outlined">
+                    search
+                    </span>
```

Comment:
Indentation could be improved. Try to align any opening tag with it's closing tag. That helps in code readability and maintainability.